src/containers/Homepage/init.ts

```diff
@@ -0,0 +1,20 @@
+import { Chain, } from '@nervos/plugin'
+import { IContainerProps, TransactionFromServer, } from '../../typings'
+
+const initState = {
```

Comment:
init state could be set in `src/initValue.ts`

Reply:
set init state in `src/initValue.ts`